src/css/sidebar.css

```diff
@@ -0,0 +1,57 @@
+.panel-container {
+    display: flex;
+    flex-direction: column;
+}
+
+.sidebar-container {
+    height: 100%;
+    background-color: white;
+    z-index: 2;
+}
+
+.sidebar-content {
+    overflow-y: auto;
+    height: calc( 100% - 52px );
```

Comment:
We should need to do any CSS calculations. This wouldn't adjust to the height changes of its siblings. You could change this to `flex: 1 1 auto` and add `display: flex` and `flex-direction: column` to `.sidebar-container` to keep things the right height.